CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to Bandit
+Thanks for considering take part in improvement of Bandit project, contributions are always welcome!  
```

Comment:
Thanks for considering **to** take part in **the** improvement of Bandit, we welcome all contributions.